tensorboard/examples/plugins/example_basic/README.md

```diff
@@ -2,51 +2,160 @@
 
 ## Overview
 
-You know (and, we hope, love!) TensorBoard’s core features like the scalar dashboard, the graph explorer, and the embedding projector. However, in every TensorBoard user’s life, there comes a time when they want some cool new visualization that just…doesn’t exist yet. That’s what the TensorBoard plugin system is for.
+You know (and, we hope, love!) TensorBoard’s core features! However, in every TensorBoard user’s life, there comes a time when you want some cool new visualization that just…doesn’t exist yet. That’s what the TensorBoard plugin system is for.
 
-This document will explain high level concepts using the example plugin and provide guidelines on plugin authorship. To get started right away, jump to the [Local Development](#local-development) section.
+This document will explain high level concepts using the example plugin and provide guidelines on plugin authorship. To get started with an example, jump to the [Getting Started](#getting-started) section.
 
 A plugin is comprised of three components:
 
   - The **Backend** is where you write Python code that does post-processing of your data and serves the data to your plugin frontend in the browser.
   - The **Frontend** is where your custom visualization lives.
-  - The optional **Summary** component is how users of your plugins will write data that your plugin can read from their TensorFlow programs.
+  - The optional **Summary** component is how users of your plugins will write data that your plugin can read from their TensorFlow programs. See [docs](https://www.tensorflow.org/api_docs/python/tf/summary) for details.
+
+The backend and frontend operate within a plugin lifecycle:
+
+  - **1) Plugin setup**: When a user starts `tensorboard --logdir ...`, TensorBoard discovers available plugins, allows them to parse command line flags if needed, and configures URL routes to be served.
```

Comment:
Nit: be consistent with the title casing. Rest are Capital Cased while this is not. Also these title feels a bit weird :\.

Reply:
I've made all the "lifecycle phase" names regular, non-title case.  Are the titles themselves okay?  I think "Web session created" sounds strange, but I couldn't think of a better name

Reply:
Haha, I wanted to comment on that but my creativity is limited and could not come up with better titles. 

`Plugin initialized`, `User loads TensorBoard`, `Dashboard opened`, ...?

Reply:
Done as
`Plugin initializes`
`User loads TensorBoard`
`User opens dashboard`
`Plugin handles routes` ?

Reply:
Sounds good!